DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,204 @@
+describe('login', () => {
+
+    const testData = require('../fixtures/testData.json')[0]
```

Comment:
you can remove [ and ] at the beginning and the end of fixture file so you can access without [0].